apiserver/plane/license/management/commands/configure_instance.py

```diff
@@ -15,6 +15,17 @@ def handle(self, *args, **options):
         from plane.license.utils.encryption import encrypt_data
         from plane.license.utils.instance_value import get_configuration_value
 
+        mandatory_keys = ["SECRET_KEY"]
+
+        for item in mandatory_keys:
+            if not os.environ.get(item):
+                self.stdout.write(
+                    self.style.ERROR(
+                        f"{item} env variable is required."
+                    )
+                )
+                exit(1)
```

Comment:
Hi @Royalsspirit, rather than using `exit` you can use a `CommandError` something like `raise CommandError(f"{item} env variable is required.")`

Reply:
Indeed, way way better than what i did. Thank you